docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+adjacent events.  It is also a special connection because the
+:math:`\text{p}_I` transition symmetry value for the two transitions are
+equal but opposite in sign.  Additionally, the :math:`\text{d}_I` transition symmetry
+remains unchanged (:math:`\Delta \text{d}_I = 0`) for the two transitions.
+(In fact, this behavior under a :math:`\pi` rotation is generally true for
+odd (:math:`\text{p}_I, \text{f}_I, \ldots)` and even (:math:`\text{d}_I, \text{g}_I, \ldots)`
+rank spin transition symmetry functions.)
+
+While a :math:`\pi/2` rotation can transfer a transition into many other transitions, it
+is useful to know that the :math:`\text{d}_I` transition symmetry value cannot remain
+unchanged (:math:`\Delta \text{d}_I \neq 0`) between two connected transitions
+under a :math:`\pi/2` rotation.
+
+Finally, another useful result is
+
+.. math::
+    :label: zeroPulseTransition
+
+    \ketbra{I,m_f}{I, m_i}  \stackrel{(0)_\phi}{\longrightarrow} \ketbra{I, m_f}{I, m_i}.
+
+While it's not surprising that a rotation through an angle of zero does nothing
+to the transition, this turns out to help act as the opposite of a total mixing
+event, i.e., a **no** mixing event. As a convenience, this is defined as a
+"NoMixing" query and can be implemented with the code below.
+
+
+.. plot::
+    :context: close-figs
+
+
+    MixingEvent(query="NoMixing"),
+
+
+The **MixingEvent** object holds the rotation details in a **MixingQuery** object as
+a **RotationQuery** object associated with a ``channels`` attribute.  This is
+illustrated in the sample code below.
+
+.. plot::
+    :context: close-figs
+
+
+    import numpy as np
+    from mrsimulator.method.query import RotationalQuery
+    rot_query = RotationalQuery(angle=np.pi/2, phase=0)
+    rot_mixing = MixingEvent(query={"ch1": rot_query})
+
+
+Frequency Contributions
+-----------------------
+
+The NMR frequency, :math:`\Omega(\Theta,i,j)`, of an :math:`i  \rightarrow  j`
+transition between the eigenstates of the stationary-state semi-classical
+Hamiltonian in a sample with a lattice spatial orientation, :math:`\Theta`, can
+be written as a sum of components,
+
+.. math::
+    \Omega(\Theta,i,j) = \sum_k \Omega_k(\Theta,i,j)
+
+with each component, :math:`\Omega_k(\Theta,i,j)`, separated into three parts:
+
+.. math::
+    \Omega_k(\Theta,i,j) = \omega_k \, {\Xi}^{(k)}_L (\Theta) \,{\xi}^{(k)}_\ell (i,j),
+
+where :math:`{\xi}^{(k)}_\ell(i,j)` are the spin transition symmetry functions
+described earlier, :math:`{\Xi}^{(k)}_L(\Theta)` are the spatial symmetry
+functions, and :math:`\omega_k` gives the size of the kth frequency component.
+The experimentalist indirectly influences a frequency component :math:`\Omega_k`
+by direct manipulation of the quantum transition, :math:`i \rightarrow  j`, and
+the spatial orientation,  :math:`\Theta` of the sample.
+
+The function symbol :math:`\Xi_\ell(\Theta)` is similarly replaced with the
```

Comment:
> The function symbol :math:`\Xi_\ell(\Theta)` is similarly replaced with the

It is probably edited from a different context, but the word *similarly* doesn't seem to fit in this sentence.